README.md

```diff
@@ -64,7 +64,15 @@ func testOutlets() {
 }
 ```
 
-#### Testing constraints
+### Testing UIKit Components
+#### UIControl
+
+```swift
+MTKAssertControl(testVC.loginButton, sends: #selector(LoginViewController.didTapLoginButton(_:)), to: testVC, for: .touchUpInside, "The login button should be hooked up to the login action.")
+ 
+```
+ 
+### Testing Constraints
```

Comment:
Should this be a subsection under UIKit components?  It feels like it could go either way.